<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RealizarReservaDialog</class>
 <widget class="QDialog" name="RealizarReservaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(234, 240, 242);</string>
  </property>
  <widget class="QDateEdit" name="checkInDateEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>280</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="numDiariasSpinBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>340</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="localidadeComboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="tipoQuartocomboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="descontoComboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>400</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="valorEstimadoLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>390</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="nomeClienteLineEdit">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>280</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Nome Cliente</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cpfClienteLineEdit">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>340</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>CPF cliente</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmarReservaButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>450</y>
     <width>94</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>confirmar</string>
   </property>
  </widget>
  <widget class="QLabel" name="valorEstimadoLabel_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="icons.qrc">:/images/imagens/logoo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>170</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 20pt &quot;Nimbus Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Dados da Reserva:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 20pt &quot;Nimbus Roman&quot;;</string>
   </property>
   <property name="text">
    <string>_________________________</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 20pt &quot;Nimbus Roman&quot;;</string>
   </property>
   <property name="text">
    <string>_________________________</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;Nimbus Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Local:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>320</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;Nimbus Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Tipo Quarto:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>380</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;Nimbus Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Desconto:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>260</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;Nimbus Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Check_In:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>320</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;Nimbus Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Di√°rias:</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
